great-idea/index.html

```diff
@@ -15,9 +15,67 @@
 </head>
 
   <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
-    
+  <header>
+  <nav class="navigation">
+    <a href="services.html">Services</a>
+    <a href="">Product</a>
+    <a href="">Vision</a>
+    <a href="">Features</a>
+    <a href="">About</a> 
+    <a href="">Contact</a> 
+  
+  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </nav>
+</header>
+  <div class="container">
+    <section class="head">
+  <h1 class="innovation">
+  Innovation<br>
+  On<br>
+  Demand
+  </h1>
+ 
+  <div class="button">Get Started</div>
+  <img src="img/header-img.png" alt="Image of a code snippet.">
+    </section>
+  <section> 
+    <div class="topContent">
+      <div class="features">
+  <h2>Features</h2>
+  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      </div>
+      <div class="about">
+  <h2>About</h2>
+  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      </div>
+    </div>
+  </section>  
+  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+ <section>
+  <div class="bottomContent">
+        <div class="item">
+  <h2>Services</h2>
+  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </div>
+        <div class="item">
+  <h2>Product</h2>
+  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </div>
+        <div class="item">
+  <h2>Vision</h2>
+  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+</section>     
+  <address class="contact">
+      <h2>Contact</h2>
+      123 Way 456 Street<br>
+      Somewhere, USA<br>
+      1 (888) 888-8888<br>
```

Comment:
```suggestion
      <a href="tel:18888888888">1 (888) 888-8888</a><br>
```
making this an anchor tag with "tel:PHONE-NUMBER" in the href, this becomes more semantic and allows for a better UX as your user's device will know that this is a phone number